src/orange/surf/LocalSurfaceVisitor.hh

```diff
@@ -157,23 +152,7 @@ CELER_FUNCTION T LocalSurfaceVisitor::get_item(Items<T> const& items,
 {
     CELER_EXPECT(*range.end() <= items.size());
     CELER_EXPECT(item < range.size());
-
-    T const* ptr = LocalSurfaceVisitor::get_ptr(items, *range.begin());
-    return *(ptr + item.unchecked_get());
-}
-
-//---------------------------------------------------------------------------//
-/*!
- * Get a pointer to the item at the given item index.
- */
-template<class T>
-CELER_FUNCTION T const*
-LocalSurfaceVisitor::get_ptr(Items<T> const& items, ItemId<T> item)
-{
-    CELER_EXPECT(item < items.size());
-
-    T const* ptr = items[AllItems<T>{}].data();
-    return ptr + item.unchecked_get();
+    return items[range][item.unchecked_get()];
```

Comment:
While you're at it, could you rename the local variable `range` to something else? (It's shadowing the `celeritas::range` function.)